src/javascript/usage-trends/components/charts/trend-chart.js

```diff
@@ -22,19 +22,19 @@ import {loadScript} from '../../../utils';
 
 // The material charts color palette.
 const COLORS = [
-    '#4285F4', '#DB4437', '#F4B400', '#0F9D58', '#AB47BC', '#00ACC1',
-    '#FF7043', '#9E9D24', '#5C6BC0', '#F06292', '#00796B', '#C2185B'];
+  '#4285F4', '#DB4437', '#F4B400', '#0F9D58', '#AB47BC', '#00ACC1',
+  '#FF7043', '#9E9D24', '#5C6BC0', '#F06292', '#00796B', '#C2185B'];
 
 
 let gvizLoadPromise = null;
```

Comment:
@diminishedprime here's an example of a `let` binding that eslint *didn't* convert to `const`, in case you were worried about that like I was